ports/bux/vcpkg.json

```
@@ -0,0 +1,18 @@
+{
+  "name": "bux",
```

```suggestion
  "name": "buck-yeh-bux",
```
`bux` is a very short name -- could you please rename the port to make it clear what library is referred to?

---

Changes pushed